src/Tools/SarifValidator/JSONError.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Microsoft.CodeAnalysis.Sarif.SarifValidator
+{
+    public enum JSONErrorKind
+    {
+        Syntax,
+        Validation
+    }
+
+    public enum JSONErrorLocation
+    {
+        InstanceDocument,
+        Schema
+    }
+
+    public class JSONError
+    {
+        public JSONErrorLocation Location { get; set; }
```

Comment:
Fixed. Note that the libraries have several types that are named "JSONXxx"; can't change that.